src/components/layout.js

```diff
@@ -39,21 +57,17 @@ const Layout = ({ children }) => (
                 'react, reactjs, JavaScript, frontend, web development, dallas',
             },
           ]}
+          title={data.site.siteMetadata.title}
```

Comment:
just curious, is the ordering significant?

Reply:
It's whatever I thought about and wrote down. We do have an open issue #17 that hopefully will encourage any SEO thought leaders to update this naive implementation.